spark/dl/src/test/scala/com/intel/analytics/bigdl/utils/tf/TensorflowLoaderSpec.scala

```diff
@@ -386,7 +469,7 @@ class TensorflowLoaderSpec extends TensorflowSpecHelper{
     tmpLocation.delete()
     tmpLocation.mkdir()
 
-    require(runPython(s"$modelScript $tmpLocation ${endPoints.mkString(",")}"),
+    require(runPython(s"$modelScript $tmpLocation ${endPoints.mkString(",")} False"),
```

Comment:
`testModelBackward` runs a forward but doesn't return the forward output to comparing. There are some redundant codes in this function.